src/modules/common/components/ScrollingSearchContainer.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
```

Comment:
the naming is maybe not optimal, because it would be a container in the components folder

Reply:
Hmm.. yeah... suggestions ? :P

Reply:
maybe just ScrollingSearchBar?